conf/spark-env.sh.template

```diff
@@ -23,6 +23,7 @@
 # Options read when launching programs locally with
 # ./bin/run-example or ./bin/spark-submit
 # - HADOOP_CONF_DIR, to point Spark towards Hadoop configuration files
+# - YARN_CONF_DIR, to point Spark towards YARN configuration files
```

Comment:
Sure, but you have added this to the section that doesn't concern YARN. Just remove this line. You added it to the YARN section below.